charts/refarch-gateway/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: refarch-gateway
 description: Helm chart for deploying the it@M refarch-gateway
 type: application
-version: 1.3.1
+version: 1.3.2-1
```

Comment:
unfortunally this version doesn't work. please make a real version 1.3.2

`helm.sh/chart: refarch-gateway-1.3.1`

https://github.com/it-at-m/helm-charts/blob/main/charts/sps-sample/Chart.yaml#L16